.github/actions/repo-gardening/src/get-files.js

```diff
@@ -0,0 +1,32 @@
+/* global GitHub */
+const debug = require( './debug' );
+
+/**
+ * Get list of files modified in PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<Array>} Promise resolving to an array of all files modified in  that PR.
+ */
+async function getFiles( octokit, owner, repo, number ) {
+	const fileList = [];
+
+	debug( 'add-labels: Get list of files modified in this PR.' );
+
+	for await ( const response of octokit.paginate.iterator( octokit.pulls.listFiles, {
+		owner,
+		repo,
+		pull_number: +number,
```

Comment:
You can reduce the number of API calls by requesting 100 per page instead of the default 30.
```suggestion
		pull_number: +number,
		per_page: 100,
```